ds/ds.php

```diff
@@ -0,0 +1 @@
+<?php
```

Comment:
Please, move code from ds_c.php to ds.php. There is no actual need in this empty file.

Reply:
Done. Also deleted duplicate classes which somehow came into the file. 🤔